src/modules/expression-evaluator/src/types.ts

```diff
@@ -22,6 +22,7 @@ export interface IParameters {
   objects?: object[]
   pgConnection?: IConnection
   graphQLConnection?: IGraphQLConnection
+  APIfetch?: any // Don't know type of fetch object
```

Comment:
I'll have a look if I can find out what type could be used here. Would probably be a combination of types, because like you said the front-end and back-end use different APIfetch obejcts.

Reply:
Looks like `APIfetch?: Function` is acceptable. 👍